ee/packages/api-client/src/index.ts

```diff
@@ -260,11 +260,19 @@ export class RestClient implements RestClientInterface {
 		const data = new FormData();
 
 		Object.entries(params as any).forEach(([key, value]) => {
-			if (value instanceof File) {
+			if (Array.isArray(value)) {
+				value.forEach((file) => {
+					if (file instanceof File) {
+						data.append(key, file, file.name);
+						return;
+					}
+					file && data.append(key, file as any);
+				});
+			} else if (value instanceof File) {
 				data.append(key, value, value.name);
-				return;
+			} else {
```

Comment:
I don't think this change is necessary anymore right? Since the upload is back to a single file per API call

Reply:
yes sir i have removed this changes